src/hooks/useMe.ts

```diff
@@ -43,4 +45,16 @@ export function useMe() {
       enabled: !!accountHeaders,
     },
   );
+
+  useEffect(() => {
+    const listener = async () => {
+      await useMeQuery.refetch();
```

Comment:
We might want to refetch the app config if we aren't already doing it. If the user is different but the project is the same, they might be in different cohorts and need to get the correct app config version

Reply:
Good point, I will look into that.

Reply:
With my recent refactor all components part of the LoggedInStack are unmounted on logout. This means app-config will be refetched every time there is a fresh login.